doc/book/box/data_model.rst

```diff
@@ -119,67 +119,93 @@ Therefore a developer often deals with two type sets:
 the types of the programming language (such as Lua) and
 the types of the Tarantool storage format (MsgPack).
 
-.. _index-box_lua-vs-msgpack:
+..  _index-box_lua-vs-msgpack:
 
 Lua versus MsgPack
 ~~~~~~~~~~~~~~~~~~
 
-.. container:: table
-
-    .. rst-class:: left-align-column-1
-    .. rst-class:: left-align-column-2
-    .. rst-class:: left-align-column-3
-    .. rst-class:: left-align-column-4
-
-    +-------------------+-------------------------+--------------------------------+------------------------------+
-    | Scalar / compound | MsgPack |nbsp| type     | Lua type                       | Example value                |
-    +===================+=========================+================================+==============================+
-    | scalar            | nil                     | "`nil`_"                       | ``nil``                      |
-    +-------------------+-------------------------+--------------------------------+------------------------------+
-    | scalar            | boolean                 | "`boolean`_"                   | ``true``                     |
-    +-------------------+-------------------------+--------------------------------+------------------------------+
-    | scalar            | string                  | "`string`_"                    | ``'A B C'``                  |
-    +-------------------+-------------------------+--------------------------------+------------------------------+
-    | scalar            | integer                 | "`number`_"                    | ``12345``                    |
-    +-------------------+-------------------------+--------------------------------+------------------------------+
-    | scalar            | float 64 (double)       | "`number`_"                    | ``1.2345``                   |
```

Comment:
I do see it's kind of offtopic for the `datetime` discussion, but still is bothering me: why there are 2 rows for double literals? (LuaJIT ffi cdata and standard Lua number). What we wanted to highlight here? (That we could, depending on value, may end up using either number or extended LuaJIT type? But similarly we would have for integer values, but there are no duplication.)

I'me very much confused here...